web-frontend/src/pages/TransferCoins/SendCoins/SendCoins.tsx

```diff
@@ -1,6 +1,36 @@
+import { Tabs, Button } from '@massalabs/react-ui-kit';
+import WalletLayout from '../../../layouts/WalletLayout/WalletLayout';
+import { useLocation } from 'react-router-dom';
+import Send from './Send';
+
 function SendCoins() {
-  // This is a placeholder
-  return <h1>Send coins </h1>;
+  const { state } = useLocation();
+  const nickname = state.nickname;
+
+  console.log(nickname);
+
+  const tabsConfig = [
+    {
+      label: 'Send',
+      content: <Send />,
+      onClickTab: () => console.log('Hello'),
+    },
+    {
+      label: 'Receive',
```

Comment:
![image](https://github.com/massalabs/thyra-plugin-wallet/assets/83134981/868659c4-2edf-486c-99d0-01848f10b00f)
so that we can switch from send to receive